docs/pages/getting-started/quick-start.md

```diff
@@ -23,6 +23,14 @@ dojoup
 
 For full `dojoup` reference and debugging see [Dojoup](/toolchain/dojoup.md).
 
+### Install scarb (optional)
+
+For the language server to work properly, your local Scarb version should not be lower than Dojo's cairo version. To accomplish this, install it with the following command:
+
+```sh
+# todo: input command here
```

Comment:
Instead of the script, we may give the user the commands to run for him to see both versions: the dojo one, and the local one.
WDYT?

Reply:
Sure. Gonna try it.

Reply:
I show the user the commands to see scarb local version and installation of scarb on commit [be0b3ed](https://github.com/dojoengine/book/pull/243/commits/be0b3ed888d8899fe0d414ab118ddc90c5fa6bc2)

I'm uncertain which command to use to display the version of Cairo used by Dojo @glihm

Reply:
> I show the user the commands to see scarb local version and installation of scarb on commit [be0b3ed](https://github.com/dojoengine/book/pull/243/commits/be0b3ed888d8899fe0d414ab118ddc90c5fa6bc2)
> 
> I'm uncertain which command to use to display the version of Cairo used by Dojo @glihm I am testing it with dojo lastest alpha 0.7 release

That's actually a good point. We should add to `sozo --version` something to output the actual cairo version that is supported by Dojo. This should be the easiest way to display such information and to ensure the user can easily install it.

Reply:
I can create a PR for that